core/app/models/concerns/spree/parameterizable_name.rb

```
@@ -11,6 +11,15 @@ module ParameterizableName
       before_validation :set_name_from_presentation, if: -> { name.blank? }
       before_validation :normalize_name
 
+      #
+      # Scopes
+      #
+      scope :search_by_name, ->(query) do
```

RuboCop (1.75.5)

[convention] Use the `lambda` method for multiline lambdas.

(Style/Lambda)